### DISCUSSION TITLE
TearDown-Attribute is not working with headless-platform for ui-testing using Nunit framework?

### DISCUSSION THREAD

**Discussion by pm-da** [AvaloniaUI/Avalonia#16751] [Q&A]
I would like to use the "TearDown" attribute when testing with Nunit. However, the "TearDown" method is never executed, even if I set a breakpoint in it during a running unit test. Is there a reason for this? The attributes “OneTimeSetUp”, “SetUp” and “OneTimeTearDown” work as they should.

**Comment by Tom-RedCactus** [comment]
Issue is created for this: https://github.com/AvaloniaUI/Avalonia/issues/17475
